AGENTS.md

```
@@ -0,0 +1,162 @@
+- Handle API errors in stores with boolean flags
+
+### Templates
+- Self-closing tags for void elements: `<VsBrDivider />`
+- Use kebab-case attributes: `:search-term`, `v-if`, `@click`
+- Props on own lines, use `v-for` with `:key`
+
+### CSS/SCSS
+- Use scoped styles: `<style scoped>`
+- Follow BEM-like naming from component library
+
+### ESLint Rules
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)